pkg/health/server/prober.go

```diff
@@ -43,8 +43,8 @@ type prober struct {
 
 	// 'stop' is closed upon a call to prober.Stop(). When the stopping is
 	// finished, then prober.Done() will be notified.
-	stop         chan bool
-	proberExited chan bool
+	stop         chan struct{}
+	proberExited chan struct{}
 	done         chan bool
```

Comment:
I think the `done` channel could be changed to `chan struct{}` as well, though it requires changing `(*prober).Done()`.

Upon closer inspection this func even looks unused, so it doesn't look like anything is checking the channel being closed, so maybe we don't need the `done` channel after all.

Reply:
Thanks...Done!